heron/healthmgr/src/java/com/twitter/heron/healthmgr/common/MissingStateException.java

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.healthmgr.common;
+
+/**
+ * This exception is generated when desired state information is not found
+ */
+public class MissingStateException extends RuntimeException {
```

Comment:
Could we change to something like `InvalidStateException` to make it slightly more generic? That way it could be used for any inconsistent or unexpected state.